GA/post.py

```
@@ -0,0 +1,40 @@
+    try:
+        # Open and load the results file
+        with results_path.open('rb') as f:
+            res = dill.load(f,
+                            ignore=False)
+
+        # Print the objective function values
+        print("Objective function values:")
+        print(res.F)
+        
+    except FileNotFoundError as e:
+        raise FileNotFoundError(f"Error: Results file not found. Ensure .dill exists: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)